nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -280,3 +280,31 @@ recoveries_server:
 robot_state_publisher:
   ros__parameters:
     use_sim_time: True
+
+bt_waypoint_follower:
+  ros__parameters:
+    use_sim_time: True
+    bt_xml_filename: "follow_waypoints.xml"
+    plugin_lib_names:
+    - nav2_compute_path_to_pose_action_bt_node
```

Comment:
These are mostly navigation specific behavior tree nodes, are you sure you want to have all these as default for an autonomy application?

Reply:
No.
I'll fix it.

Reply:
I've updated `plugin_lib_names`.